src/lib/UtapiClient.js

```diff
@@ -131,82 +180,65 @@ export default class UtapiClient {
     * @return {object} this - current instance
     */
     pushMetric(metric, reqUid, params, cb) {
+        assert(methods[metric], `${metric} metric is not handled by Utapi`);
```

Comment:
Pushing a metric that Utapi cannot handle will cause the API's callback to never be called, as it throws an `AssertionError`. If you were testing it, then this assertion would be handy for indicating the problem—it will tell you which metric is unable to be pushed. The same applies for object properties expected for each API's metric, and the property types. Utapi is user-friendly! 😺